kombu/common.py

```diff
@@ -50,7 +50,11 @@ def get_node_id():
 def generate_oid(node_id, process_id, thread_id, instance):
     ent = bytes_if_py2('%x-%x-%x-%x' % (
         node_id, process_id, thread_id, id(instance)))
-    return str(uuid3(NAMESPACE_OID, ent))
+    try:
+        ret = str(uuid3(NAMESPACE_OID, ent))
+    except:
```

Comment:
Bare except statements should be avoided as they are too broad as they do not handle only the intended exception(s).

In your case it should be changed to:

```
except ValueError:
```